apps/web/lib/d/[link]/[slug]/getServerSideProps.tsx

```diff
@@ -127,6 +127,7 @@ async function getUserPageProps(context: GetServerSidePropsContext) {
         queryDuration,
         eventData.length
       ),
+      durationConfig: eventData.metadata?.multipleDuration ?? [],
```

Comment:
Why are we introducing this?

Reply:
@anikdhabal I added this due to this bug mentioned - https://github.com/calcom/cal.com/pull/15896#issuecomment-2297672751

in this commit - https://github.com/calcom/cal.com/pull/15896/commits/0cc66d8e92c8603db7ddef8026010bcc6d14ffdb

Long PR 😅 things are becoming hazy..